### DISCUSSION TITLE
database did not come up: unable to add CA to cert pool

### DISCUSSION THREAD

**Discussion by devalexandre** [encoredev/encore#1010] [Help] [answered]
When I run encore run display Creating PostgreSQL database cluster... Failed: failed to start cluster: database did not come up: unable to add CA to cert pool

**Docker version 24.0.7, build afdd53b4e3**

![image](https://github.com/encoredev/encore/assets/4489558/751066ff-5b43-4e7f-83d7-513a14491042)

**Comment by marcuskohlberg** [comment, marked as answer]
This was already answered in the [community Slack](https://encore.dev/slack), but posting the answer here too for visibility:

> This error is commonly caused by the presence of the file $HOME/.postgresql/root.crt on the filesystem. When this file is present the PostgreSQL client library will assume the database server has that root certificate, which will cause the above error.
> To fix: Remove or rename the file, then try encore run again.

Also added to docs [here](https://encore.dev/docs/primitives/databases#troubleshooting).
